jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/utils/PayloadCreator.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.homeoffice.digital.sas.jparest.utils;
+
+import java.util.List;
+
+public class PayloadCreator {
+
+    public static String createIDPayload(List<String> fieldNameList, List<String> fieldValueList){
+        StringBuilder payload = new StringBuilder("{\n");
+        for (int i = 0; i < fieldNameList.size() && i < fieldValueList.size(); i++){
```

Comment:
consider JSONObject class we talk about and let me know if that was useful for your solution, Laurie. Many thanks

Reply:
Done, I've converted this method to use Gson